packages/react/src/Footnotes/Footnotes.examples.stories.tsx

```diff
@@ -34,7 +34,11 @@ export const RiversWithCitations = () => (
                       Third-party tools support your favorite languages and frameworks{' '}
                       <Text size="100" weight="light">
                         <sup>
-                          <InlineLink href="#economic-impact" id="economic-impact-ref">
+                          <InlineLink
+                            href="#economic-impact"
+                            id="economic-impact-ref"
+                            style={{display: 'inline-block', minWidth: 'var(--base-size-24)'}}
```

Comment:
The space is a bit too wide IMO, and doesn't work when you have text following the reference link. 

<img width="355" height="201" alt="Screenshot 2025-07-10 at 11 52 14" src="https://github.com/user-attachments/assets/7dfa3e0d-16ca-42e0-83c4-b64791fac9f6" />

Could we knock this back to `--base-size-12`?

Also, we aren't adding any space vertically or to the left of the link, is that okay? I assumed increasing click target meant adding space in all directions?

Reply:
There's not any specific guidance on how/where we should apply the space in this particular case. If we were to add additional vertical space then we'd either be increasing the line height, or at risk of having our touch area overlap with nearby links.

We don't actually _have_ to increase the touch area here as it's an inline link and exempt from that accessibility criterion. This is purely a recommendation to improve UX

Reply:
We could go to a min-height of 24px by the looks of it, as that's the line height we use in this scenario